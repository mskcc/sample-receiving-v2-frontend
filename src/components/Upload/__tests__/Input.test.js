import React, { Component } from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'
import TestInput from '../formComponents/Input'

const Input = props => <TestInput {...props} />
const onChange = jest.fn();


describe('Render Input', () => {
  it('render Input correctly', () => {
    const InputComponent = renderer.create(<Input onChange={onChange} />).toJSON()
    expect(InputComponent).toMatchSnapshot()
  })
})
