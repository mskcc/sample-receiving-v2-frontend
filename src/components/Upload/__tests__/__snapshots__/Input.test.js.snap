// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Input render Input correctly 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root Input-textField-1"
>
  <label
    className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
    data-shrink={false}
    htmlFor="test_me"
  >
    Missing translationId: upload.form.test_me_label for language: \${ languageCode }
  </label>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-describedby="test_me-helper-text"
      aria-invalid={false}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      id="test_me"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="number"
    />
  </div>
  <p
    className="MuiFormHelperText-root"
    id="test_me-helper-text"
  >
    Missing translationId: upload.form.test_me_helptext for language: \${ languageCode }
  </p>
</div>
`;
